specification/storage/resource-manager/Microsoft.Storage/stable/2021-01-01/storage.json

```diff
@@ -2472,12 +2528,22 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "None",
+            "SystemAssigned",
+            "UserAssigned",
+            "SystemAssigned,UserAssigned"
           ],
           "x-ms-enum": {
             "name": "IdentityType",
-            "modelAsString": false
+            "modelAsString": true
           }
+        },
+        "userAssignedIdentities": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/UserAssignedIdentity"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the set of User Assigned identities that will be used with this storage account. The key is the ARM resource identifier of the identity."
```

Comment:
As we discussed , only 1 identity should be add.
we might should add the information to the description.

Reply:
Updated

Reply:
Your description contradicts with your definition, your description says that you can get or set but your definition has readOnly: true properties which means a user won't be able to set these values.

---
In reply to: [569133331](https://github.com/Azure/azure-rest-api-specs/pull/12807#discussion_r569133331) [](ancestors = 569133331)